### PR TITLE
couple permissions fixes for Sightings

### PR DESCRIPTION
**Pull request by naknomum**
## Pull Request Overview

- Rules for reading a Sighting were overly strict
- Note: this has some tiny changes related to anon submissions but this needs more work re: public-user-ownership so they are being held off for now.

**Pull Request Checklist**
- [x] Ensure that the PR is properly formatted
  - Example: All lint checks are passing
- [x] Ensure that the PR is properly rebased
  - Example: The PR is rebased on `develop` (commit: `<insert develop commit hash>`)
- [x] Ensure that the PR uses a consolidated database migration
  - Example: One database migration is proposed (revision `<insert develop revision>` -> `<insert new revision>`)
- [x] Ensure that the PR is properly tested
  - Example: All automated tests are passing
- [x] Ensure that the PR is properly covered
  - Example: The percentage of the code covered by tests has not decreased
- [x] Ensure that the PR is properly sanitized
  - Example: No sensitive data or large content was added to this PR
- [ ] Ensure that the PR is properly reviewed

### CODE REVIEW

File: tests/modules/sightings/resources/test_create_sighting.py [hunk 1]
```diff
@@ -82,9 +82,57 @@ def test_create_and_delete_sighting(db, flask_app_client, researcher_1, staff_us
...
     # upon success (yay) we clean up our mess
     sighting_utils.cleanup_tus_dir(transaction_id)
     sighting_utils.delete_sighting(flask_app_client, researcher_1, sighting_id)
 
     post_ct = test_utils.multi_count(db, (Sighting, Encounter, Asset, Submission))
     assert orig_ct == post_ct
+
+
+def test_create_anon_and_delete_sighting(db, flask_app_client, staff_user):
+    return  # DISABLING TEST FOR NOW
```
Comment (hwindsor) [inline tests/modules/sightings/resources/test_create_sighting.py:99]: crafty.
The other way to do this is to change the function name, if it doesn't start with test_ pytest doesn't run it

**Review by hwindsor — APPROVED** [review]
LGTM

File: tests/modules/sightings/resources/test_create_sighting.py (hunk 1 above)
Comment (naknomum) [reply tests/modules/sightings/resources/test_create_sighting.py:99]: i have wondered about that!   thx.

**Comment by naknomum** [comment]
needs /rebase

**Pull request merged**